aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -614,17 +614,17 @@ Tensor& linalg_solve_sparse_csr_out(const Tensor& input, const Tensor& other, Te
     TORCH_CHECK(singularity == -1, "Probably got a singular matrix as an input. Please check the input again.");
     return result;
   } else {
-    TORCH_CHECK(false, "PyTorch was not built with CUDA support. Please rebuild again with USE_CUDA=1.");
+    AT_ERROR("PyTorch was not built with CUDA support. Please use PyTorch built CUDA support");
```

Comment:
Please, do not use `AT_ERROR`, it's deprecated `TORCH_CHECK(false,` was correct here.
https://github.com/pytorch/pytorch/blob/e1b84e1b6bd688c7b3bff3e7742c1f8875966e6e/c10/util/Exception.h#L630-L633

Reply:
Sorry, I wasn't aware of this. Thanks, this has been fixed now.